specs/required-inputs/README.md

```diff
@@ -0,0 +1,66 @@
+# Required Input Controls
+
+## Problem Statement
+
+Input controls are often used in forms or property editor dialogs. Often the user is required to provide an input to the control before the form can be submitted or before the property changes can be saved.
+
+We need to provide a standardized way for the application to let the end-user know that a control is required.
+
+Initially, we need this for:
```

Comment:
For completeness I'll note that [`aria-required` is also supported for `tree` and `tree-grid` roles](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-required#associated_roles). We don't support labels on either our tree or table component yet but we do have some clients that treat them as more complex selector inputs. I could imagine them wanting to force a user to make a selection and indicate that via an asterisk too.

Might be useful to explicitly note that those components are excluded for scope reasons, not because we forgot to consider them.

Reply:
Added a note in the problem statement.